app/src/main/res/values/strings.xml

```diff
@@ -415,6 +415,7 @@
     <string name="double_amount">%1$.2f</string>
     <string name="double_and_string">%1$.2f %2$s</string>
     <string name="string_and_string">%1$s %2$s</string>
+    <string name="string_and_string_share">%1$s%2$s</string>
```

Comment:
Yeah good catch but we can't name it with share, I would suggest make it specially for share link. 

<string name="playstore_link">%1$s%2$s</string>

something.